README.md

```diff
@@ -47,9 +47,9 @@ The `STANDARD` authentication mode is the default mode, so it is not necessary t
 Common configuration parameters:
 
 - `fieldsToExtract`:
-  - keys represent names of user's properties, which will be stored in JWT body / UserInfo.additionalInfo
+  - keys represent names of user's properties, which will be stored in JWT body / UserInfo.user
```

Comment:
Breaking change. Would it be possible to make it back-compatible? i.e. at least preserve `additionalInfo` name